ocean_lib/models/bfactory.py

```diff
@@ -33,7 +35,9 @@ def newBPool(self, from_wallet: Wallet) -> str:
 
         # grab pool_address
         warnings.filterwarnings("ignore")  # ignore unwarranted warning up next
```

Comment:
Please remove this line.  The warnings don't need to be ignored like this.  `errors=DISCARD` already prevents them from occuring.